apps/docs/src/index.data.ts

```diff
@@ -0,0 +1,249 @@
+          { name: "Timepicker", implemented: isImplemented("OnyxTimepicker") },
+          { name: "Calendar", implemented: isImplemented("OnyxCalendar") },
+          { name: "Accordion", implemented: isImplemented("OnyxAccordion") },
+          { name: "Slider", implemented: isImplemented("OnyxSlider") },
+          { name: "Stepper", implemented: isImplemented("OnyxStepper") },
+          { name: "Upload", implemented: isImplemented("OnyxUpload") },
+        ],
+      },
+      {
+        id: "expansion-4",
+        label: "Expansion 4",
+        description: "Low priority components.",
+        components: [
+          { name: "Breadcrumb", implemented: isImplemented("OnyxBreadcrumb") },
+          { name: "Table of Content", implemented: isImplemented("OnyxTableOfContent") },
+          { name: "Wizard", implemented: isImplemented("OnyxWizard") },
+          { name: "Tabs", implemented: isImplemented("OnyxTabs") },
+        ],
+      },
+    ];
+
+    return {
+      componentCount: watchedFiles.length,
+      variantCount,
+      mergedPRCount,
+      closedIssueCount,
+      contributorCount,
+      timestamp: timestamp.toUTCString(),
+      downloads,
+      packageCount: packageFolders.length,
+      componentTabs,
+    };
+  },
+});
+
+/**
+ * Counts the occurrences of the word in the given content.
+ */
+const countWord = (content: string, word: string): number => {
+  return content.split(word).length - 1;
+};
+
+/**
+ * Executes the given GitHub search query and returns the total_count of matched items.
+ *
+ * @see: https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#about-search
+ */
+const searchGitHub = async (
+  endpoint: "issues" | "commits",
+  filterString: string,
+): Promise<number> => {
+  // see: https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#constructing-a-search-query
+  const queryString = encodeURIComponent(`repo:SchwarzIT/onyx ${filterString}`);
+
+  // since we only need the total_count, we can decrease the per_page to 1 to improve request speeds
+  const response = await fetch(
+    `https://api.github.com/search/${endpoint}?q=${queryString}&per_page=1`,
+    {
+      headers: {
+        "X-GitHub-Api-Version": "2022-11-28",
+      },
+    },
+  );
+  const body = await response.json();
+
+  if (response.status < 200 || response.status >= 300) {
+    throw new Error(`GitHub request failed. Response body: ${JSON.stringify(body)}`);
+  }
+
+  if (typeof body !== "object" || typeof body.total_count !== "number") {
+    throw new Error("GitHub search response does not contain total_count");
+  }
+
+  return body.total_count;
+};
+
+/**
+ * Gets the number of contributors.
+ *
+ * @see: https://docs.github.com/en/rest/metrics/statistics?apiVersion=2022-11-28#get-all-contributor-commit-activity
+ */
+const getGitHubContributorCount = async (): Promise<number> => {
+  // since we only need the total_count, we can decrease the per_page to 1 to improve request speeds
```

Comment:
the URL does not have per_page -> is it missing in the fetch or is the comment wrong?

Reply:
The comment is outdated, I changed the API request :) I removed it